ibm/service/power/ibm_pi_constants.go

```diff
@@ -206,12 +207,13 @@ const (
 	Attr_Images                                      = "images"
 	Attr_ImageType                                   = "image_type"
 	Attr_InputVolumes                                = "input_volumes"
+	Attr_InstanceID                                  = "instance_id"
 	Attr_Instances                                   = "instances"
 	Attr_InstanceSnapshots                           = "instance_snapshots"
 	Attr_InstanceVolumes                             = "instance_volumes"
 	Attr_IOThrottleRate                              = "io_throttle_rate"
 	Attr_IP                                          = "ip"
-	Attr_IPAddress                                   = "ipaddress"
+	Attr_IPAddress                                   = "ip_address"
```

Comment:
I see this is used in `pi_network_port`. Why are we changing the name now?

Reply:
I have updated it.